nameserver/zone.go

```diff
@@ -18,6 +18,7 @@ var rdnsDomainLen = len(RDNSDomain) + 1
 
 type Zone interface {
 	AddRecord(ident string, name string, ip net.IP) error
+	DeleteRecords(ident *string, ip *net.IP, name *string) (count int)
```

Comment:
Yes, I'm sure there is a historic reason for returning the error... maybe it is time to get rid of it...